cranelift/codegen/src/isa/aarch64/inst/mod.rs

```diff
@@ -2216,10 +2216,11 @@ impl Inst {
                     VecALUOp::Fcmeq => ("fcmeq", size),
                     VecALUOp::Fcmgt => ("fcmgt", size),
                     VecALUOp::Fcmge => ("fcmge", size),
-                    VecALUOp::And => ("and", VectorSize::Size8x16),
-                    VecALUOp::Bic => ("bic", VectorSize::Size8x16),
-                    VecALUOp::Orr => ("orr", VectorSize::Size8x16),
-                    VecALUOp::Eor => ("eor", VectorSize::Size8x16),
+                    VecALUOp::And => ("and", size.as_scalar8_vector()),
```

Comment:
I know it's pre-existing but could we leave a comment here why we special-case the size? IIRC, it's because that's the only supported case, and for bitwise operators it doesn't matter what we consider the lane divisions?